terminology/Ballerina.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+org = "ballerinax"
+name = "health.fhir.r4.terminology"
+version = "1.0.0"
+distribution = "2201.6.0"
+authors = ["Ballerina"]
+keywords = ["Healthcare", "FHIR", "R4", "Terminology"]
+repository = "https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4"
+export = ["health.fhir.r4.terminology"]
+
+[[dependency]]
+org = "ballerinax"
+name = "health.fhir.r4"
+version = "3.0.1"
+repository = "local"
+
+[[dependency]]
+org = "ballerinax"
+name = "health.fhir.r4.international401"
+version = "1.0.0"
+repository = "local"
```

Comment:
missing EoF

Reply:
Updated in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/9678d7ff4364ca21388770b102d0029dafdc76b5